### PR TITLE
fix: fix MA SDK Github link

### PR DESCRIPTION
**Pull request by justin-fiedler**
# Amplitude Developer Docs PR

[AMP-82462](https://amplitude.atlassian.net/browse/AMP-82462)

Update MA SDK Github link to use `v1.x` branch
https://github.com/amplitude/Amplitude-TypeScript/tree/v1.x/packages/marketing-analytics-browser

[AMP-82462]: https://amplitude.atlassian.net/browse/AMP-82462?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by justin-fiedler** [comment]
Does anyone know if there is an easy way to ignore Megalinter for a single line? This warning is invalid:

image

**Review by Mercy811 — APPROVED** [review]
There is another one on line 13. Please change that one too. 🙏

**Comment by Mercy811** [comment]
Have no idea how to disable megalinter for a single line. Maybe just force merge? This file was merged probably before we set up megalinter.

File: docs/data/sdks/marketing-analytics-browser/index.md [hunk 1]
```diff
@@ -100,11 +100,11 @@ The following information is tracked in the page view events.
...
-|`event_properties.page_domain`| `string`. The page domain. | location.hostname or ''. |
-|`event_properties.page_location`| `string`. The page location. | location.href or ''. |
-|`event_properties.page_path`| `string`. The page path. | location.path or ''.|
-|`event_properties.page_title`| `string`. The page title. | document.title or ''.|
-|`event_properties.page_url`| `string`. The value of page url. | location.href.split('?')[0] or ``.|
+|`event_properties.page_domain`| `string`. The page domain. | `location.hostname` or ''. |
+|`event_properties.page_location`| `string`. The page location. | `location.href` or ''. |
+|`event_properties.page_path`| `string`. The page path. | `location.path` or ''.|
+|`event_properties.page_title`| `string`. The page title. | `document.title` or ''.|
+|`event_properties.page_url`| `string`. The value of page url. | `location.href.split('?')[0]` or ``.|
```
Comment (Mercy811) [inline docs/data/sdks/marketing-analytics-browser/index.md:107]: 👍 Nice to know that if megalinter doesn't check code blocks

**Review by bgiori — APPROVED** [review]

**Review by yuhao900914 — APPROVED** [review]

**Pull request merged**